mmcv/runner/hooks/evaluation.py

```diff
@@ -124,13 +157,21 @@ def _init_rule(self, rule, key_indicator):
 
         if rule is None:
             if key_indicator != 'auto':
-                if key_indicator in self.greater_keys:
+                # use case-insensitive key indicator to infer the rule
+                key_indicator_lc = key_indicator.lower()
```

Comment:
indicate that `lc` is the abbrev. for `lower case` in the comments.

Reply:
done.